smooth/framework/functions/plot_results.py

```diff
@@ -1,4 +1,5 @@
 from matplotlib import pyplot as plt
+from smooth.framework.simulation_parameters import SimulationParameters
 from smooth.framework.functions.functions import extract_flow_per_bus
 from smooth.examples.example_plotting_dicts import comp_dict_german, bus_dict_german, y_dict_german
 
```

Comment:
Line 13,14 should be   
:param smooth_result: ...
:type smooth_result: ...

instead of 
  :param smooth_results: ...
  :param smooth_results: ...

Reply:
Fixed in https://github.com/rl-institut/smooth/pull/182